CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+## [2.1.2]
+### Added
+### Changed
+### Deprecated
+### Removed
+### Fixed
+- Fixed Async Client without http_auth argument ([#297](https://github.com/opensearch-project/opensearch-py/pull/297))
```

Comment:
@liran-cohen-hs The next version of opensearch-py will be 2.2.0 not 2.1.2. So, I suggest without mentioning the version number just write it under unreleased. I mean replace [2.1.2] with [Unreleased].

Reply:
makes sense :)
updated
thanks